pybest/preproc.py

```diff
@@ -79,10 +79,10 @@ def _run_func_parallel(ddict, cfg, run, func, logger):
     if 'fs' in cfg['space']:  # assume gifti
         if cfg['iscifti'] == 'y':
             data, tr = load_and_split_cifti(func, cfg['atlas_file'],cfg, cfg['left_id'], cfg['right_id'], cfg['subc_id'], cfg['mode'])
-            tr /= 1000 # defined in msec
+            #tr /= 1000 # defined in msec
         else:
             data, tr = load_gifti(func, cfg, return_tr=True)
-            tr /= 1000  # defined in msec
```

Comment:
The gifti part is not mine. I think gifti does return TR in msec and needs the /= 1000

Reply:
At the end after checking, it turns out that taking out completely these line was fine as it has to come out in seconds and load_gifti does good at that. So i'm now not changing the tr at all and it works just fine. Sorry for the confusion.